### PR TITLE
ci: 21.14 image

### PR DESCRIPTION
**Pull request by joeldodge79**

### CODE REVIEW

File: .github/workflows/python-ci.yml [hunk 1]
```diff
@@ -94,10 +94,10 @@ jobs:
         os:
           - ubuntu
         looker:
-          - '21_6'
           - '21_8'
```
Comment (jkaster) [inline .github/workflows/python-ci.yml:97]: Isn't 21.6 still the extended support release?

File: .github/workflows/tssdk-ci.yml [hunk 2]
```diff
@@ -98,9 +98,10 @@ jobs:
         os:
           - ubuntu
         looker:
-          - '21_6'
           - '21_8'
```
Comment (jkaster) [inline .github/workflows/tssdk-ci.yml:101]: 21_6 or 21_8?

**Review by jkaster — APPROVED** [review]
Looks good as long as 21.8 is the correct version instead of 21.6

File: .github/workflows/python-ci.yml (hunk 1 above)
Comment (joeldodge79) [reply .github/workflows/python-ci.yml:97]: oh interesting, I didn't realize the ESR pattern. According to https://docs.looker.com/relnotes/supported-releases 21.6 ESR support ends today (as does 21.8) so the list should actually only have 21_10, 21_12, 21_14

**Review by jkaster — APPROVED** [review]
GTG

**Pull request merged**